tokens/manage/multisig-safe.md

```diff
@@ -351,4 +351,45 @@ contract SetText {
 
 这样就意味着您已经成功使用Safe钱包与智能合约交互了！
 
+## 使用Moonbeam Safe API {: #using-moonbeam-safe-apis }
+
+Moonbeam、Moonriver和Moonbase Alpha也支持用API来与Moonbeam Safe进行交互。
+
+=== "Moonbeam"
+     ```
+     {{networks.moonbeam.multisig.api_page }}
+     ```
+
+=== "Moonriver"
+     ```
+     {{networks.moonriver.multisig.api_page}}
+     ```
+
+=== "Moonbase Alpha"
+     ```
+     {{networks.moonbase.multisig.api_page}}
+     ```
+
+作为使用API的示例，请尝试从Moonbeam Safe API检索有关多签钱包的信息。从Safe页面，复制您的多签地址：
+
+![Contract Interaction Call Result](/images/tokens/manage/multisig-safe/safe-30.png)
+
+然后您可以开始使用API：
+
+ 1. 从上面的Multisig API表中打开对应的网络页面
```

Comment:
```suggestion
 1. 打開對應網絡的API頁面
```

@hyd628 I used google translate for this one line so I can just merge this PR. If it's wrong, can you please fix it? 🙂

Reply:
haha thanks. It's good, but in traditional Chinese. I changed it to simplified Chinese to be consistent in the xcm PR.

Reply:
haha oops! thank you!! 🙌